docs/tutorials/gcp_cpu.md

```diff
@@ -0,0 +1,247 @@
+      ```
+
+      Run:
+
+      ```
+      gcloud compute ssh [YOUR_INSTANCE_NAME]
+      ```
+
+      Either command works for the purpose of this tutorial. Continue to step 4 then
+      complete the steps below, regardless of which command you use.
+
+      **1. Copy the container ID for your qsim Docker container**
+
+      Use the `docker ps` command to display the container ID. The output should look
+      like the following text:
+
+      ```
+      CONTAINER ID IMAGE COMMAND CREATED STATUS PORTS NAMES i
+      8ab217d640a3 gcr.io/quantum-291919/jupyter_qsim:latest "jupyter-notebook --…" 2 hours ago Up 2 hours 0.0.0.0:8888->8888/tcp dazzling_lovelace.
+      ```
+
+      In this case, the container ID is `8ab217d640a3`.
+
+      **2. Connect to your qsim Docker container**
+
+      Use the docker exec command line tool to login to your container. Replace
+      `[CONTAINER_ID]` with the ID that you copied in step 1.
+
+      ```
+      docker exec -it [CONTAINER_ID] /bin/bash
+      ```
+
+      Your command prompt now executes commands in the container.
+
+      **3. Verify your installation**
+
+      You can use the code below to verify that qsim uses your qsim installation.
+      You can paste the code directly into the REPL, or paste the code in a
+      file.
+      ```
+      # Import Cirq and qsim
+      import cirq
+      import qsimcirq
+
+      # Instantiate qubits and create a circuit
+      q0, q1 = cirq.LineQubit.range(2)
+      circuit = cirq.Circuit(cirq.H(q0), cirq.CX(q0, q1))
+
+      # Instantiate a simulator that uses the GPU
+      qsim_simulator = qsimcirq.QSimSimulator()
+
+      # Run the simulation
+      print("Running simulation for the following circuit:")
+      print(circuit)
+
+      qsim_results = qsim_simulator.compute_amplitudes(
+          circuit, bitstrings=[0b00, 0b01])
+
+      print("qsim results:")
+      print(qsim_results)
+      ```
+
+      After a moment, you should see a result that looks similar to the following.
+
+      ```
+      [(0.7071067690849304+0j), 0j]
+      ```
+
+## Clean up
+
+After you finish either tutorial, you can avoid continued billing by stopping or
+deleting the VM instance that you create; visit the
+[Compute Instances dashboard](https://pantheon.corp.google.com/compute/instances)
+to manage your VM.
+
+For more information about managing your VM, see the following documentation
+from Google Cloud:
+
+*   [Stopping and starting a VM](https://cloud.google.com/compute/docs/instances/stop-start-instance)
+*   [Suspending and resuming an instance](https://cloud.google.com/compute/docs/instances/suspend-resume-instance)
+*   [Deleting a VM instance](https://cloud.google.com/compute/docs/instances/deleting-instance)
+
+## Next steps
+
+After you finish, don't forget to stop or delete your VM on the Compute
+Instances dashboard. For more information, see the
+[Clean Up section in the Overview](https://quantumai.google/qsim/tutorials/gcp_overview#clean_up).
```

Comment:
The section this links to doesn't appear to exist?

Reply:
Fixed.